api/src/test/java/org/openmrs/CohortMembershipTest.java

```diff
@@ -252,4 +252,73 @@ public void compareTo_shouldSort() throws Exception {
 		
 		assertThat(list, contains(startedRecently, startedLongAgo, noStartOrEnd, endedRecently, endedLongAgo, voided));
 	}
+	
+	@Test
+	public void equal_shouldReturnTrueIfObjectReferenceAreTheSame() {
+		CohortMembership cohortMembership = new CohortMembership(12);
+		
+		boolean result = cohortMembership.equals(cohortMembership);
```

Comment:
I guess we could just assert without the temporary variable.

Reply:
I thought it is more OpenMRS agree on style to first execute the function under test and then perform the assertion? Anyways I changed it